comms/storage/telemetry/logger.go

```diff
@@ -0,0 +1,11 @@
+package telemetry
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+)
+
```

Comment:
For now this will just have `zerolog` stuff but ideally this will fully wrap any lib we use. I can add a `Logger` interface in the future